backend/Bindings/Edna.Bindings.User/UsersClient.cs

```diff
@@ -0,0 +1,36 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Edna.Utils.Http;
+using Newtonsoft.Json;
+
+namespace Edna.Bindings.User
+{
+    public class UsersClient
+    {
+        private readonly string _baseServiceUriString;
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        internal UsersClient(string serviceUrlEnvironmentVariable, IHttpClientFactory httpClientFactory)
```

Comment:
I'm a bit confused on why we are passing the `serviceUrlEnvironmentVariable` and not creating `const` in this class itself 🤔.

Reply:
Or thinking the other way around, why are we not passing the `serviceUrl` itself in the arguments?

Reply:
I believe these explorations can be kept as good first issues for now and be utilized to help onboard someone new in the project. What do you suggest?

Reply:
I believe this would again be something copy-paste from other bindings and might be taken up later on.